File: beginner/c/1188.c
#include <stdio.h>

int main (){

  double m[12][12], soma = 0.0;
  int i, j, cont = 0, aux = 5, coluna = 7;
  char operacao;

  scanf(" %c", &operacao);

  for(i = 0; i < 12; i++){
    for(j = 0; j < 12; j++){
      scanf("%lf", &m[i][j]);
    }
  }

  for(i = 7; i < 12; i++){
    for(j = aux; j < coluna; j++){
        soma += m[i][j];
        cont++;
    }
    coluna++;
    aux--;
  }

  if(operacao == 'S'){
    printf("%.1lf\n", soma);
  } else{
    printf("%.1lf\n", soma/cont);
  }


}

File: beginner/python/1188.py
operacao = input()

m = [[0 for j in range(12)] for i in range(12)]

for i in range(12):
    for j in range(12):
        m[i][j] = float(input())

def soma_or_media(mat, op):
    coluna = 7
    cont = 0
    soma = 0.0
    aux = 5

    for i in range(7,12):
        for j in range(aux, coluna):
            soma += mat[i][j]
            cont += 1
            
        coluna+=1
        aux-=1

    if op == 'S':
        return soma
    else:
        return soma/cont

print("%.1f" % (soma_or_media(m, operacao)))